### ISSUE TITLE
github action: unit test error

### ISSUE THREAD

**Issue by JustinJaniszewski** [ubsuny/23-Homework5G3#48]
Your github action unit test references a "tests.py" file but there isn't any.

**Comment by uarif** [comment]
Yes, I also noticed that last night, but I guess, That is because that member just added a raw code because the unit test file was not available till then.

**Comment by s4il3sh** [comment]
> Your github action unit test references a "tests.py" file but there isn't any.

We haven't started yet any unittesting.
